.github/workflows/config.yml

```diff
@@ -0,0 +1,33 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
```

Comment:
This will be the name that will be used on GitHub Actions interface. Names make make it easier for us to understand what job does what. The name could be: `Build docker image`

Reply:
I just want to say that the name of the file is also too generic. The name could be `Tests` and the file `tests.yaml` or `pr_tests.yaml` and then call the job `build-docker`. This way people will see:

Tests / build-docker  -  Fail